xblockmufi/public/view.less.min.css

```diff
@@ -1 +0,0 @@
ground-color:#00d!important}.xblockmufi_block .xblockmufi_block_textarea .toolbar li:hover a,.xblockmufi_block .xblockmufi_block_textarea .toolbar li.specialButton.active a{background-color:#444;color:#fff}.xblockmufi_block .xblockmufi_block_textarea .toolbar li.specialButton{float:right;border-right:none;border-left:1px solid #ccc}.xblockmufi_block .xblockmufi_block_textarea .toolbar li.specialButton a{padding:6px 8px 3px 8px}.xblockmufi_block .xblockmufi_block_textarea .toolbar li.specialButton i{line-height:15px;font-size:24px;vertical-align:baseline}.xblockmufi_block .xblockmufi_block_textarea .toolbar li.specialButton .sr .toggle-open{display:none}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable{display:none;position:relative;float:left;clear:both;width:100%;font-family:"Andron Scriptor Web",serif!important}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charZoomTitle{height:22px;padding:4px 120px 2px 5px;text-align:right;line-height:22px;border-bottom:1px solid #ccc}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charZoomTitle span{font:400 1em "Andron Scriptor Web"}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charZoom{width:100px;height:100px;padding:5px;float:right;border-left:1px solid #ccc;font-size:60px;line-height:1.75em;text-align:center;background:#fff}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charZoom .intro-zoom-icon{margin-top:19px}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charWrapper{float:right;margin-left:-1px}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charWrapper .char{width:29px;height:30px;padding:5px;margin-top:-1px;margin-right:-1px;float:right;border:1px solid #ccc;font-size:22px;line-height:1.5em;text-align:center}.xblockmufi_block .xblockmufi_block_textarea .specialCharTable .charWrapper .char:hover{cursor:pointer;background-color:#444;color:#fff}.xblockmufi_block .xblockmufi_block_textarea .contents{position:relative;top:-1px;padding:10px;min-height:250px;clear:both;overflow:auto;resize:vertical;border-top:1px solid #ccc;font-family:"Andron Scriptor Web",serif!important}.xblockmufi_block .xblockmufi_block_textarea b{float:left;font-size:72px;line-height:60px;padding:5px 10px 5px 0}.xblockmufi_block .xblockmufi_block_textarea li.red-text a{color:#d00}.xblockmufi_block .xblockmufi_block_textarea li.red-text:hover a{background-color:#d00!important}.xblockmufi_block .xblockmufi_block_textarea li.green-text a{color:#080}.xblockmufi_block .xblockmufi_block_textarea li.green-text:hover a{background-color:#080!important}.xblockmufi_block .xblockmufi_block_textarea li.blue-text a{color:#00d}.xblockmufi_block .xblockmufi_block_textarea li.blue-text:hover a{background-color:#00d!important}.xblockmufi_block .xblockmufi_block_textarea .hint{color:#666}.xblockmufi_block .xblockmufi_block_textarea .sr{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}
```

Comment:
@caesar2164 Ah! The reason we didn't get this error with the comma before, even though it was like that in the `view.less` file... is because we were minifying the code and the extra comma was removed! Now that we're not explicitly minifying it, the extra comma sticks around and silently errors out :(